pkg/volume/aws_ebs/aws_ebs_test.go

```diff
@@ -118,9 +135,9 @@ func TestPlugin(t *testing.T) {
 	if builder == nil {
 		t.Errorf("Got a nil Builder")
 	}
-
+	volPath := path.Join(tmpDir, "pods/poduid/volumes/kubernetes.io~aws-ebs/vol1")
```

Comment:
minor nit (doesn't really need a fix either), but a better name for this probably would've been "expectedPath".